ckanext/canada/logic.py

```diff
@@ -70,11 +72,11 @@ def wrapper(context, data_dict,
 
 @side_effect_free
 def changed_packages_activity_list_since(context, data_dict):
-    '''Return the activity stream of all recently added or changed packages.
+    '''Return the package_id and timestamp of all recently added or changed packages.
 
     :param since_time: starting date/time
 
-    Limited to 31 records (configurable via the
+    Limited to 63 records (configurable via the
```

Comment:
this query is only returning unique package ids and timestamps now. I think we can make this limit much, much larger (I'd guess in the thousands). Would be interesting to experiment with our live site and large time ranges to choose a better values.